op-program/host/config/config.go

```diff
@@ -49,7 +49,7 @@ var (
 )
 
 type Config struct {
-	L2ChainID eth.ChainID // TODO: Forbid for interop
+	L2ChainID eth.ChainID
```

Comment:
why aren't these parameters an issue for interop? are they just ignored?

Reply:
Yeah we wound up enforcing that they aren't used elsewhere (most importantly in the local preimage provider).  I'm not sure I meant to commit those todos in the first place, I was originally just working out what the different config requirements were.